seed/landing/views.py

```diff
@@ -157,8 +157,8 @@ def create_account(request):
             ''' End reCAPTCHA validation '''
             if result['success']:
                 user = form.save(commit=False)
+                user.username = user.username.lower()
                 user.is_active = False
-                user.save()
```

Comment:
Things don't seem to be working, and I suspect it's from this line. Should we still be saving/creating the user Object here?

Reply:
🙏  Thanks for catching this, I'm not sure how that got deleted...

Reply:
I actually don't see the modal shift happening anymore 👍